### ISSUE TITLE
Threads in channels not showing up 

### ISSUE THREAD

**Issue by iokobi** [hfaran/slack-export-viewer#138]
Just noticed that there are no replies or conversation threads showing up. 
Is this a bug or am I doing something wrong during the export?

**Comment by hdsheena** [comment]
I see that too, though not sure if its the export or not yet

**Comment by hdsheena** [comment]
Yeah, my data doesnt have the threads, so hoping I can address it from that side!

**Comment by etcy** [comment]
I am using the newest ver 1.1.
Threads show up at the end of the page, rather than in chronological order alongside messages. 
This makes it impossible to make sense of the contents.

Anything I can help to debug this problem?

**Comment by etcy** [comment]
I did further testing with 1.5 years of export. 
Some of the thread replies do appear in the right date order. 
But some of it simply shows up at the end of the page. I have not been able to find a pattern to this logic.

**Comment by devianvii** [comment]
@etcy hotfix for our problem: https://github.com/devianvii/slack-export-viewer

**Comment by devianvii** [comment]
@iokobi @hdsheena It is most likely that your problem is related to https://github.com/hfaran/slack-export-viewer/issues/121.
Starting from some version, Slack stopped including threads in messages.
How do you export?
